Integrations/GitHub/GitHub.py

```diff
@@ -671,6 +672,28 @@ def get_commit_command():
     return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
 
 
+def list_commits(author):
+    params = {'author': author} if author else {}
+    return http_request('GET', url_suffix=f'{USER_SUFFIX}/commits',
+                        params=params)
+
+
+def list_commits_command():
+    args = demisto.args()
+    response = list_commits(author=args.get('author'))
+    limit = args.get('limit')
+    if limit:
+        response = response[:int(limit)]
+    commits = [format_commit_outputs(c['commit']) for c in response]
```

Comment:
We are getting `null` in the SHA path in the context for every commit. This is because in every object in this response is different than the object returned when you fetch one commit.
The `SHA` key of the commit is not under the `commit` key but rather not he root of the object.
So you will have to create a different `format_commit_outputs` to create the outputs correctly, or to mutate the objects before you send it to `format_commit_outputs`
this is the raw response of the commits:
![lala - Artifact Viewer 2019-12-09 17-37-53](https://user-images.githubusercontent.com/19407287/70449582-e52e4a00-1aaa-11ea-9b83-1177b55785c0.png)